index.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+	plugins: [
+		'add-module-exports',
+		'@babel/plugin-syntax-dynamic-import',
+		['@babel/plugin-proposal-class-properties', { loose: true }],
```

Comment:
So, the discussion is mostly about this.
I think we need to define the defaults.

- I've used `@babel/plugin-proposal-class-properties` on some places, but in the end this causes reactivity with vue, so I'm wondering if that is really worth it.
- `@babel/plugin-syntax-dynamic-import` is really useful to split bundles and async load them
- @PVince81 is `add-module-exports` necessary for all?